src/lang/en-US.ts

```diff
@@ -1259,6 +1260,18 @@ const FieldSelection: ResolvedIntlConfig['messages'] = {
     'fieldSelection.table.label.unknown': `Unknown`,
 };
 
+const Notifications: ResolvedIntlConfig['messages'] = {
+    'notifications.paymentMethods.missing.title': `Missing Payment Method`,
+    'notifications.paymentMethods.missing.cta': `add a payment method`,
+    'notifications.paymentMethods.missing.cta.alreadyThere': `add a payment method below`,
+    'notifications.paymentMethods.missing.trialCurrent': `The free trial for {tenant} ends in {daysLeft} days, but no payment method has been added to your account.`,
+    'notifications.paymentMethods.missing.trialCurrent.instructions': `Please {cta} before your trial ends to continue using Estuary Flow.`,
+    'notifications.paymentMethods.missing.trialEndsToday': `The free trial for {tenant} ends today, but no payment method has been added to your account`,
+    'notifications.paymentMethods.missing.trialEndsToday.instructions': `Please {cta} today to continue using Estuary Flow.`,
+    'notifications.paymentMethods.missing.trialPast': `{tenant} is past it's free trial without a payment method.`,
```

Comment:
> 'notifications.paymentMethods.missing.trialPast': `{tenant} is past it's free trial without a payment method.`

There is a typo in this piece of content: `it's` should be replaced by `its`, the possessive form of _it_.

Reply:
fixed